Kudu.Core/Scripts/selectLatestVersion.ps1

```diff
@@ -0,0 +1,17 @@
+﻿PARAM(
```

Comment:
That would get you the version of whatever go you're running. The goal is to find the latest go version among the various versions we have on the box. So it comes down to searching through folders.